index.html

```diff
@@ -25,15 +25,140 @@
+                        </ul>
+                    </div>
+                    <div class="flex-item4"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                    <div class="flex-item5">
+                        <ul>
+                            <li class="html"><a href="#">html</a></li>
+                            <li class="css"><a href="#">css</a></li>
+                            <li class="javaScript"><a href="#">javaScript</a></li>
+                        </ul>
+                    </div>
+                    <div class="flex-item6"><button>See project</button></div>
+                 </div>
+                 <div class="grid-item">
+                    <div class="flex-item1"><img src="./images/Snapshoot4.png" alt="my tonic project"></div>
+                    <div class="flex-item2"><h1>Multi-Post<br> Stories</h1></div>
+                    <div class="flex-item3"> 
+                        <ul>
+                            <li><a href="#" class="canopy">CANOPY</a></li>
+                            <li><a href="#"><img src="./images/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">Back End Dev</a></li>
+                            <li><a href="#"><img src="./images/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">2015</a></li>
+                        </ul>
+                    </div>
+                    <div class="flex-item4"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+                    <div class="flex-item5">
+                        <ul>
+                            <li class="html"><a href="#">html</a></li>
+                            <li class="css"><a href="#">css</a></li>
+                            <li class="javaScript"><a href="#">javaScript</a></li>
+                        </ul>
+                    </div>
+                    <div class="flex-item6"><button>See project</button></div>
+                 </div>
+           </div>
+        </section>
+        <section class="about-myself">
+            <h1>About <br> Myself</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <P class="connect">LET'S CONNECT</P>
+            <ul class="contact-icon">
+            <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+            </ul>  
+            <button>Get my resume</button>     
```

Comment:
- [ ] Please change the `border-radius` of this button to `8px` to match the figma design

Reply:
I have applied border-radius: 8px.